doc/ref_model/chapters/chapter04.md

```diff
@@ -21,25 +26,139 @@ The idea of the infrastructure profiles catalogue is to have a predefined set of
-|-----------|------|-------|------------|----------------------|
-| .tiny | 1 | 512MB | 1GB | 1Gbps |
-| .small | 1 | 2GB | 40GB | 1Gbps |
-| .meduim | 2 | 4GB | 40GB | 1Gbps |
-| .large | 4 | 8GB | 80GB | 1Gbps |
-| .large2* | 4 | 16GB | 80GB | 1Gbps |
-| .xlarge* | 8 | 16GB | 160GB | 1Gbps |
-| .xlarge2* | 8 | 32GB | 160GB | 1Gbps |
-| .xlarge3* | 8 | 64GB | 160GB | 1Gbps |
+Flavours represent the compute, memory, storage capacity, and management network resource templates that are used to create the VMs on the compute hosts. Each VM instance is given a flavour (resource template), which determines the instance’s core, memory and storage characteristics. 
 
-<p align="center"><b>Table 4-1:</b> Compute flavours.</p>
+Flavours can also specify secondary ephemeral storage, swap disk, metadata to restrict usage, or special project access. In other terms, Flavours are grouping of hosts typically sharing same characteristics or metadata. 
+
+A compute flavour geometry consists of the following elements:
+
+Element |Description 
+--------|----------
+Name	|A descriptive name
+Virtual compute resources (aka vCPUs) |Number of virtual compute resources (vCPUs) presented to the instance.
+Memory MB	|Instance memory in megabytes. 
+Ephemeral/Local Disk |Specifies the size of an ephemeral data disk that exists only for the life of the instance. Default value is 0.<br />The ephemeral disk may be partitioned into boot (base image) and swap space disks. 
+Is Public	|Boolean value, whether flavor is available to all users or private to the project it was created in. Defaults to True.
+Extra Specs	|Key and value pairs that define on which compute nodes a flavor can run. These pairs must match corresponding pairs on the compute nodes. Use to implement special resources, such as flavors that run on only compute nodes with GPU hardware.
+
+<p align="center"><b>Table 4-1:</b> Flavour Geometry Specification.</p>
+
+Flavour Capabilities:
+
+1. **CPU Oversubscription Ratio**: is based on the number of threads available. For example, on a 24-core host with HT, there are 48 vCPUs with 1:1 CPU Ratio and 96 vCPUs with 2:1 CPU Ratio.
+
+2. **HT (Hyper Threading support)**: Enabled on all servers. Gets 2 hyper threads per physical CPU.  Always ON. 
+3. **CPU Pinning**: vCPU is pinned to a physical core and dedicated to the requesting VM. 
+4. **NUMA (Non-Uniform Memory Access) Alignment**: Indicates that vCPU will be on a Socket that is aligned with the associated NIC card and memory.  Important for performance optimized VNFs. 
+5. **Huge Pages**: By default, CPU allocate RAM in 4K chunks. Huge Pages enable to allocate in larger Chunks (such as 2MB, 1GB).  This helps improve performance in some cases. 
+6. **SR-IOV (Single-Root Input/Output Virtualization)**: Allows SR-IOV ports to be used in VM. 
+7. **DPDK vRouter**: Virtual Router integrated with the Intel DPDK (Data Plane Development Kit) libraries. 
+8. **Kernel vRouter**: Virtual Router that runs in the Kernel space
```

Comment:
What is the correct terminology here ? virtual router or virtual switch ? I think we need to replace router term with switch

Reply:
The industry name is vSwitch , vRouter is vendor naming  (Juniper , Nuage and E//) which vRouter is an enhancement  vSwitch + DPDK